### DISCUSSION TITLE
今天一天删了88个版本

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#3187] [讨论]
看到PCL2内 光不常用版本就96个 难受死了 下定决心删除了所有不想玩的版本 一共88个
其中的大头是平时发的一些快照，我下了就是图个新鲜和跟上时代 根本不玩

**Comment by ghost** [comment]
em，1.20总共也就20多个快照，你是怎么做到88个的，我就2个服务器端mc，愚人节5个，1.20.1有3，1.16.5，1.8.9，1.7.2

**Reply by iPodtouch6557** [reply]
从1.17追到1.20，追了三年了

**Comment by WorldHim** [comment]
话说这种内容不太适合发 GitHub 上吧…

**Reply by iPodtouch6557** [reply]
好的 以后注意

**Comment by iPodtouch6557** [comment]
> em，1.20总共也就20多个快照，你是怎么做到88个的，我就2个服务器端mc，愚人节5个，1.20.1有3，1.16.5，1.8.9，1.7.2



> 从1.17追到1.20，追了三年了
